apps/studio/components/interfaces/Auth/AuthProvidersForm/ProviderForm.tsx

```diff
@@ -54,7 +54,12 @@ const ProviderForm = ({ config, provider }: ProviderFormProps) => {
       const isDoubleNegative = doubleNegativeKeys.includes(key)
 
       if (provider.title === 'SAML 2.0') {
-        initialValues[key] = (config as any)[key] ?? false
+        const configValue = (config as any)[key]
+        initialValues[key] = configValue
+          ? configValue
+          : provider.properties[key].type === 'boolean'
+            ? false
+            : ''
```

Comment:
this might be better?
`initialValues[key] = configValue || (provider.properties[key].type === 'boolean' ? false : '')`

Reply:
Good catch

Reply:
Can be for later but we can do the same with the lines right below it